### PR TITLE
feat: Add Prime React

### PR DESCRIPTION
**Pull request by vinaysanwal**
## Changes proposed

1) Added New UI Library Card Prime React  under UI Library 

## Screenshots

Screenshot 2024-10-07 at 1 18 51 PM

### CODE REVIEW

**Comment by rupali-codes** [comment]
@vinaysanwal please attach the closing issue.

**Comment by vinaysanwal** [comment]
> @vinaysanwal please attach the closing issue.

I have added a new feature in UI library card attached screenshot as well

**Comment by vinaysanwal** [comment]
close #2525

File: layouts/GeneralLayout.tsx
Comment (rupali-codes) [inline layouts/GeneralLayout.tsx:1]: hi @vinaysanwal, please remove comments

**Comment by rupali-codes** [comment]
Closing the PR due to inactivity

**Pull request closed without merging**